test/functional/apps/console/_autocomplete.ts

```diff
@@ -84,6 +84,58 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
     });
 
+    describe('with aggregation templates', async () => {
+      const AGGREGATION_TEMPLATES = [
```

Comment:
I targeted this PR just to handle the `geo_*` aggregations, but any of the supported agg templates could be added to this array.